legacy/src/app/partials/nodedetails/storage/disks-partitions.html

```diff
@@ -1176,4 +1176,4 @@ <h3 class="p-heading--four">Used disks and partitions</h3>
 
 
 <p>Learn more about deploying <a href="https://docs.maas.io/en/installconfig-images" class="p-link--external" target="_blank">Windows</a></p>
-<p>Change the default layout in <a href="/MAAS/settings/storage/">Settings &rsaquo; Storage</a></p>
+<p>Change the default layout in <a href="/MAAS/r/settings/storage">Settings &rsaquo; Storage</a></p>
```

Comment:
Same as the other PR. Is it possible to use the env vars here to add the "/MAAS" and "/r" parts? In ./legacy/.env there should be two variables:
```
BASENAME="/MAAS"
REACT_BASENAME="/r"
```

Reply:
@huwshimi No, they don't work in the HTML templates

Reply:
Nevermind - I found a way :)